plugins/smart_proxy_ansible/changelog

```diff
@@ -1,3 +1,15 @@
+ruby-smart-proxy-ansible (3.2.1-1) stable; urgency=low
+
+  * 3.2.1 released
+
+ -- Adam Ruzicka <aruzicka@redhat.com>  Wed, 23 Jun 2021 15:56:38 +0200
+
+ruby-smart-proxy-ansible (3.2.0-1) stable; urgency=low
+
+  * 3.2.0 released
+
+ -- Adam Ruzicka <aruzicka@redhat.com>  Wed, 23 Jun 2021 15:04:21 +0200
+
```

Comment:
do we want to include this? it will not be in repository, just in the history.... :thinking:

Reply:
usually we do, yes

Reply:
even tho this version was never added in develop branch? just for a brief period in this PR? interesting :)